client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/EmbeddedCompatContinuousQueryTest.java

```diff
@@ -128,7 +128,6 @@ public void testDisallowGroupingAndAggregation() {
       continuousQuery.addContinuousQueryListener(query, listener);
    }
 
-   @Test(enabled = false, description = "Disabled due to https://issues.jboss.org/browse/ISPN-6730")
    public void testContinuousQuery() {
```

Comment:
So https://issues.jboss.org/browse/ISPN-6730 is solved by this PR or did it stop failing earlier?